k8s/namespaces/fees-pay/ccpay-callback-function/demo.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-callback-function
+  annotations:
+    fluxcd.io/automated: "true"
+    #fluxcd.io/tag.java: glob:pr-112-*
+    fluxcd.io/tag.nodejs: glob:demo
```

Comment:
```suggestion
    fluxcd.io/tag.function: glob:demo
```

Reply:
Also, it needs to be pr-* image. not demo

Reply:
Thanks for the review, modified the config